### ISSUE TITLE
Solazan Basin - Frenzyheart Tribe rep bug

### ISSUE THREAD

**Issue by nina3a2** [dalaranwow/dalaran-wow#5309]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: I wanted to start raising rep with the frennzyheart tribe at SB.. But when i went at the camp I couldn't do any Qs.. I google it and i found out that I should make a quest called "Part Time Hunter" which wants me to kill an NPC named Pitch.. Although I had already make that quest and I'm pretty sure that i didn't kill that Pitch... don't know what went wrong but now i'm stuck on neautral with frenzyheart and unfriendly with oracle.. 

**Expected behaviour**: I suppose that after completing that quest  I would be able to start quests on the Frenzyheart Hill camp.

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by galgnu** [comment]
You probably needs to do further quests afterwards. I had no problems and on my way to exalted. 
Last quest in the chain makes you kill an elite and meanwhile select between the two factions by killing one of two mobs. Afterwards you get honored by one faction and hostile towards the other. 
Look up the quest on wowhead for more information. I found all i needed thete.
